assets/scss/_styles_project.scss

```diff
@@ -1,9 +1,369 @@
-.nav-logo {
+img.height-50 {
+    height: 4   50px;
+}
+
+.td-navbar .navbar-brand span:last-child {
+    display: none;
+}
+
+
+.navbar-brand img {
+   display: inline-block;
+}
+
+.navbar-brand {
     display: inline-block;
-    margin: 0 0;
-    height: 42px;
+}
+
+
+.td-navbar .navbar-brand svg {
+    width:  160px;
+    height: auto;
+    margin: 0 20px;
+
+}
+
+.lead-block-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 2rem 1rem;
+    background-color: #081f2c;
+    gap: 2rem;
+}
+
+
+.lead-block-left {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    gap: 0.5rem;
+}
+
+.lead-block-center {
+    width: 150px;
+    margin-right: 20.5rem;
+    height: auto;
+}
+
+.lead-block-center img {
+    width: 450px;
+    height: auto;
+}
+
+.left-head-title-other {
+    font-size: 62px;
+    font-family: Noto Sans;
+    box-sizing: border-box;
+    font-style: normal;
+    font-weight: 800;
+    text-align: left;
+    line-height: 100%;
+    letter-spacing: 0.62px;
+    margin-top: 0;
+    gap: 0.25rem;
+    margin-bottom: 1rem;
+    display: grid;
+    color: #28384b;
```

Comment:
UX/accessibility issue - this color does not have strong enough contrast with the `.lead-block-container` background color.